config/initializers/devise.rb

```diff
@@ -75,7 +75,7 @@
   # Notice that if you are skipping storage for all authentication paths, you
   # may want to disable generating routes to Devise's sessions controller by
   # passing skip: :sessions to `devise_for` in your config/routes.rb
-  config.skip_session_storage = [:http_auth]
+  config.skip_session_storage =  %i[http_auth jwt]
```

Comment:
just curious, where did you find that you needed to use `jwt` here? I only found `token_auth` as another option to this config.